digital_cheatsheet.py

```
@@ -22,19 +24,24 @@ def __init__(self):
         self.hotkeysEnabled = True
         self.image_directory = tempfile.mkdtemp()
 
-        self.modelNames = ["gemini-1.5-flash", "gemini-1.5-pro"]
+        self.modelNames = ["gemini-2.5-flash", "gemini-2.5-pro"]
+
+        # Track pressed keys for hotkey combinations
+        self.pressed_keys = set()
+        self.listener = None
 
         if GEMINI_API_KEY:
```

Pylint (3.3.7)

[error] Possibly using variable 'GEMINI_API_KEY' before assignment

(E0606)